tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -2472,7 +2473,7 @@ SpirvInstruction *SpirvEmitter::doCallExpr(const CallExpr *callExpr,
             dyn_cast<CXXMethodDecl>(operatorCall->getCalleeDecl())) {
       QualType parentType =
           QualType(cxxMethodDecl->getParent()->getTypeForDecl(), 0);
-      if (isUserDefinedRecordType(astContext, parentType)) {
+      if (hlsl::IsUserDefinedRecordType(parentType)) {
```

Comment:
So this will not cover the `BuiltInTriangleIntersectionAttributes`, even though that should be treated like any other ordinary struct type.

Reply:
Ok, nevemind, in this case, `BuiltInTriangleIntersectionAttributes` can't have methods, so it can't be parentType.